ais/io.py

```diff
@@ -0,0 +1,84 @@
+"""
+Objects for reading and parsing NMEA from a file.
+"""
+
+
+import codecs
+import sys
+
+import six
+
+import ais.nmea_queue
+
+
+def open(name):
+
+    """
+    Open a file containing NMEA and instantiate an instance of `NMEAFile()`.
+
+    Args:
+      name: A file path, file-like object, or '-' for stdin.
+    """
```

Comment:
Switch to Google style.  No blank like before doc string.  No newline between """ and first line.  Needs Raises and Returns sections.